### ISSUE TITLE
Failed Bus to Threed References the wrong memory location

### ISSUE THREAD

**Issue by tragicmanner** [Lyrositor/CCScriptWriter#6]
When given the option to get off the bus near the tunnel to Threed the game references another location in memory instead of continuing onto the Yes No option.

Label is l_0xc78394 in data_32

**Comment by Lyrositor** [comment]
Do you have any more information you could provide? What happens exactly when you get the option to get off the bus?

**Comment by tragicmanner** [comment]
So, I found a *temporary* fix for this issue. As quoted from the pkhack forums:
"Take the “Yes” line out of the end of data_32.ccs and add it as the first line of l_0xc7841e in data_33.ccs. Then, at the last line of the block of l_0xc78394 in data_32.ccs, append goto(data_33.l_0xc7841e). This seems to jump over the odd memory issue and makes things behave as they should."

I made a video of the bug. Sorry the quality is pretty bad, but it should help illustrate what's going on.
http://youtu.be/dgVnC4Mh6UY
